services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/events.py

```diff
@@ -362,46 +361,49 @@ async def action(cls, app: FastAPI, scheduler_data: SchedulerData) -> None:
         )
 
         async def _pull_outputs_and_state():
-            async with disabled_directory_watcher(
-                dynamic_sidecar_client, dynamic_sidecar_endpoint
-            ):
-                tasks = [
-                    dynamic_sidecar_client.pull_service_output_ports(
+            # while outputs are being disallow the watcher from generating
```

Comment:
- Ohh dear the non English on that one.
- the `watcher` will be enabled once it is required to do so. By default the sidecar start it disabled. This could also removed from here at this point.

Reply:
- Ohh dear the non English on that one.
- the `watcher` will be enabled once it is required to do so. By default the sidecar start it disabled. This could also removed from here at this point.